src/components/Task/index.jsx

```diff
@@ -1,23 +1,45 @@
+import { TaskInfo } from './TaskInfo';
 import React from 'react';
 import PropTypes from 'prop-types';
 
 // Styled
-import { TaskSC } from './style';
+import { TaskSC, TitleSC, TagsSC, StatusSC } from './style';
+// Components
+import Tags from '../DetailsViewBody/Tags';
+import Status from '../Status';
 
 
 
-const Task = ({ changeOpenedTaskId, id, title }) => {
+const Task = ({
+    changeOpenedTaskId,
+    openedTaskId,
+    task: { id, tags, todo, title, assignees, status, created }
+}) => {
     return (
-        <TaskSC data-id={id} onClick={() => changeOpenedTaskId(id)}>
-            {title}
+        <TaskSC
+            data-id={id}
+            isOpened={openedTaskId === id}
+            onClick={() => changeOpenedTaskId(id)}
+        >
+            <TagsSC>
+                <Tags tags={tags} />
+            </TagsSC>
+            <StatusSC>
+                <Status status={status} />
+            </StatusSC>
+            <TitleSC>{title}</TitleSC>
+            <TaskInfo assignees={assignees} todo={todo} created={created} />
```

Comment:
- use boolean (`hasTodo`) instead of passing an array
- move creation of the string to this component instead of passing an array